src/bootstrap.rs

```diff
@@ -152,32 +148,20 @@ e.g. https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GettingSta
 "
     );
 
-    // Step 1. Create tables
+    // Step 1. create tables
     prepare_table(&cx, "Movie", vec!["year,N", "title,S"].as_ref()).await;
 
-    // Step 2. Download & unzip data. The sampledata.zip contains 4 files.
-    let url =
-        "https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/samples/moviedata.zip";
-    let download_dir: tempfile::TempDir = download_and_extract_zip(url).await?;
-    let content = read_to_string(download_dir.path().join("moviedata.json"))?;
-    /*
-    moviedata.json (103494 lines, 3.5M bytes)
-    The JSON file is not a DynamoDB style JSON, but standard JSON format like below:
-    [
-        {
-            "year": 2013,
-            "title": "Rush",
-            "info": {
-                "directors": ["Ron Howard"],
-                "release_date": "2013-09-02T00:00:00Z",
-                "rating": 8.3,
-                "genres": [
-                    "Action",
-    */
-
-    // Step 3. wait tables to be created and in ACTIVE status
+    // Step 2. wait tables to be created and in ACTIVE status
     wait_table_creation(&cx, vec!["Movie"]).await;
 
+    // Step 3. decompress data
+    let compressed_data = include_bytes!("./resources/bootstrap/moviedata.json.br");
+    let cursor = Cursor::new(compressed_data);
+    let mut decompressor = Decompressor::new(cursor, 4096);
```

Comment:
As in [this example](https://github.com/dropbox/rust-brotli#decompression-usage), set the buffer size to 4096